 2009-11-24  Ralf Wildenhues  <Ralf.Wildenhues@gmx.de>
 
+	Enable symbol versioning with the GNU gold linker.
+	* libltdl/m4/libtool.m4 (_LT_LINKER_SHLIBS): Accept 'GNU gold'
+	in the version string.
+	* NEWS, THANKS: Update.
+	Report by Peter Fritzsche.
+

+  - Symbol versioning works with the GNU gold linker now.
 
diff --git a/THANKS b/THANKS
index 004dbd8..c8acd67 100644
--- a/THANKS
+++ b/THANKS
@@ -140,6 +140,7 @@
   Paul Biggar			paul.biggar@gmail.com
   Paul Eggert			eggert@twinsun.com
   Peter Eisentraut		peter_e@gmx.net
+  Peter Fritzsche		peter.fritzsche@gmx.de
   Peter Jeremy			peterjeremy@optushome.com.au
   Peter Kjellerstedt		peter.kjellerstedt@axis.com
   Rainer Emrich			r.emrich@de.tecosim.com
diff --git a/libltdl/m4/libtool.m4 b/libltdl/m4/libtool.m4
index 65dd027..9abd147 100644
--- a/libltdl/m4/libtool.m4
+++ b/libltdl/m4/libtool.m4
@@ -4339,6 +4339,7 @@ dnl Note also adjust exclude_expsyms for C++ above.
     fi
     supports_anon_versioning=no
     case `$LD -v 2>&1` in
+      *GNU\ gold*) supports_anon_versioning=yes ;;
       *\ [[01]].* | *\ 2.[[0-9]].* | *\ 2.10.*) ;; # catch versions < 2.11
       *\ 2.11.93.0.2\ *) supports_anon_versioning=yes ;; # RH7.3 ...
       *\ 2.11.92.0.12\ *) supports_anon_versioning=yes ;; # Mandrake 8.2 ...
